learn/filtering_and_sorting/geosearch.mdx

```diff
@@ -21,9 +21,9 @@ Due to Meilisearch allowing malformed `_geo` fields in the following versions (v
 
 ## Preparing documents for location-based search
 
-In order to start filtering and sorting documents based on their geographic location, you must make sure they contain a valid `_geo` field.
+In order to start filtering and sorting documents based on their geographic location, you must make sure they contain a valid `_geo` or `_geojson` field.
```

Comment:
This sentence kinda makes it feel like we can sort with GeoJSON when we can't in reality. Not sure how we could reformulate 🤔